internal/duplo.go

```diff
@@ -128,3 +128,17 @@ func OutputDuploCreds(creds *DuploCredsOutput) {
 	os.Stdout.Write(json)
 	os.Stdout.WriteString("\n")
 }
+
+func PingDuploCreds(creds *DuploCredsOutput, host string) error {
+	client, err := duplocloud.NewClient(host, creds.DuploToken)
+	if err != nil {
+		return err
+	}
+
+	_, cerr := client.AdminGetInfrastructure("default")
```

Comment:
we can never use an admin only API in a place where non admins are accessing the system.  Instead, try using the API that I suggested:  `/admin/GetTenantsForUser`

Reply:
I've found the GetTenantFeatures v3/features/tenant/{id} API works and have made the code changes to utilize that.